src/ontology/mondo-edit.obo

```diff
@@ -308367,6 +308369,7 @@ xref: UMLS:C0410204 {source="MONDO:notFoundInDiseaseSubset", source="Orphanet:16
 xref: UMLS:C3645536 {source="MONDO:equivalentTo", source="Orphanet:169186"}
 is_a: MONDO:0016191 {source="Orphanet:169186"} ! qualitative or quantitative defects of titin
```

Comment:
what is the relationship to this term? should it not be a subclass?

Reply:
Agreed - I added this as a superclass